README.md

```diff
@@ -325,6 +327,24 @@ HLTV.getPlayerRanking({startDate: '2018-07-01', endDate: '2018-10-01'}).then(res
 
 ***
 
+### getEvents
+
+Parses the info from the `hltv.org/events` page
+
+Option | Type | Default value | Description |
+:---:|:---:|:---:|:---:|
+size | [EventSize](https://github.com/gigobyte/HLTV/blob/master/src/enums/EventSize.ts)? | - | Event size type. (EventSize.Small, EventSize.Big). Default (empty) combines both.
+
+```javascript
+HLTV.getEvents({}).then(res => {
```

Comment:
Since eventSize is optional, passing an empty object is a waste. It's better to make the settings optional and default the argument to `{}` like in [other endpoints](https://github.com/gigobyte/HLTV/blob/master/src/endpoints/getTeamRanking.ts#L12)

Reply:
Please check if the optional parameter is assigned correctly

Reply:
The default argument is correct, but instead of defaulting size to `''` it's better to specify the type:
```
async ({ size }: { size?: EventSize } = {})
```
I believe this will work, please try it out.

Reply:
It works!